Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM        node:5.11.0
```

Comment:
@zhangweijie-cn use 5.6.0 please to keep consistent with our other node processes.